modules/distr-tracing-tempo-config-default.adoc

```diff
@@ -4,147 +4,58 @@
 
 :_mod-docs-content-type: REFERENCE
 [id="distr-tracing-tempo-config-default_{context}"]
-= Default configuration options
+= Introduction to TempoStack configuration parameters
 
 The `TempoStack` custom resource (CR) defines the architecture and settings to be used when creating the {TempoShortName} resources. You can modify these parameters to customize your {TempoShortName} implementation to your business needs.
 
-.Example of a generic Tempo YAML file
+.TempoStack CR example
 [source,yaml]
 ----
-apiVersion: tempo.grafana.com/v1alpha1
-kind: TempoStack
-metadata:
-  name: name
-spec:
-  storage: {}
-  resources: {}
-  storageSize: 200M
-  replicationFactor: 1
-  retention: {}
+apiVersion: tempo.grafana.com/v1alpha1 # <1>
+kind: TempoStack # <2>
+metadata: # <3>
+  name: <name> # <4>
+spec: # <5>
+  storage: {} # <6>
+  resources: {} # <7>
+  replicationFactor: 1 # <8>
+  retention: {} # <9>
   template:
-      distributor:{}
-      ingester: {}
-      compactor: {}
-      querier: {}
-      queryFrontend: {}
-      gateway: {}
+      distributor:{} # <10>
+      ingester: {} # <11>
+      compactor: {} # <12>
+      querier: {} # <13>
+      queryFrontend: {} # <14>
+      gateway: {} # <15>
+  limits: # <16>
+    global:
+      ingestion: {} # <17>
+      query: {} # <18>
+  observability: # <19>
+    grafana: {}
+    metrics: {}
+    tracing: {}
+  search: {} # <20>
+managementState: Managed # <21>
```

Comment:
I'll copy the info from https://github.com/grafana/tempo-operator/pull/711/files